package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ordermentum/eslint-config-ordermentum",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
This is an odd one conceptually - it'll likely be a breaking change for any existing codebases using this right?